tests/test_crowdstrike_panther_pipeline.py

```diff
@@ -78,10 +79,60 @@ def test_basic(mock_click):
                             "KeyPath": "ParentBaseFileName",

```

Comment:
This keypath should be `"event.ParentBaseFileName"` as well.

Reply:
Should it? We have separate [transformation for process creation](https://github.com/panther-labs/pySigma-backend-panther/blob/main/sigma/pipelines/panther/crowdstrike_panther_pipeline.py#L47-L58) events and its result doesn't include `event`

Reply:
Yes, according to the mapping doc they both should start with `event.`